docs/content/docs/install/bitbucket_cloud.md

```diff
@@ -129,13 +128,17 @@ $ tkn pac create repo
         # key: “provider.token“
 ```
 
+{{< hint info >}}
+Please note that the workspace name is different from the username. To view your username in the Bitbucket UI, click on the settings icon, navigate to Personal Bitbucket Settings, and you will find your username there.
```

Comment:
```suggestion
Please note that the workspace name is different from your username. To view your username in the Bitbucket UI, click on the settings icon, navigate to the
Personal Bitbucket Settings, and there you will find your username there.
```

it would be nice to have a screenshot..